ArjunaJTA/jms/src/main/java/org/jboss/narayana/jta/jms/ConnectionProxy.java

```diff
@@ -75,53 +92,106 @@ public void close() throws JMSException {
         }
     }
 
+    /**
+     * Delegate to {@link #xaConnection}
```

Comment:
I see, I agree with you. Just I think that explicitly mentioning that method just delegates call to somewhere else brings information value. I would personally leave it here.